### PR TITLE
Add the weighted average attendance to indicator 

### PR DESCRIPTION
**Pull request by yukinko-iwasaki**
Added the weighted attendance data from [GLD](https://globaldatalab.org/areadata/table/2010/lprimary+uprimary+lsecondary+usecondary/?levels=1+4).

Data quality might be a concern. There seems to be a lot more missing values from this data source than the constructed education index. 

Here is the % of missing values for our countries in focus. 
Screenshot 2024-06-11 at 12 27 47 PM

[here](https://github.com/user-attachments/files/15792406/export.2.csv) is the quality check result for the all countries. 

**I have not saved the dataset to our storage yet.

### CODE REVIEW

**Comment by weilu** [comment]
@yukinko-iwasaki what's the missing values's base year range? Is it from 1990 to the current year of 2024 for every country?

**Comment by yukinko-iwasaki** [comment]
@weilu 
Yes it is from 1990 to current. 

I only just filtered out the years before 2010, and here is the result (slightly better for some countries
)
image
[export (3).csv](https://github.com/user-attachments/files/15794935/export.3.csv)

**Comment by weilu** [comment]
LGTM. Please uncomment the save command before merge.

File: human_development/global_data_lab_hid_transform_load_dlt.py [hunk 1]
```diff
@@ -80,8 +80,9 @@ def global_data_lab_hd_index():
             'country_name',
             'adm1_name',
             'year',
```
Comment (yukinko-iwasaki) [inline human_development/global_data_lab_hid_transform_load_dlt.py:82]: @weilu 

My understanding is that [global_data_lab_hid_transform_load_dlt.py](https://github.com/dime-worldbank/mega-indicators/pull/3/files#diff-f6c9a389a673e087bb1960852e585c20d532ab2e0d9746d1f9b9c1bd7eb4522e) takes "indicator_intermediate.global_data_lab_hd_index" as an input and overwrite ""indicator.global_data_lab_hd_index".

Where do we call this function?

Comment (weilu) [reply human_development/global_data_lab_hid_transform_load_dlt.py:82]: This is written to the target table by the DLT workflow "[Subnational Human Development Index](https://adb-6102124407836814.14.azuredatabricks.net/pipelines/a52621f7-9847-4893-aea5-d7390e29cb63?o=6102124407836814)". You can see in the settings it's pointing to the "indicator" as the target schema. The table name is as defined in the method annotation attribute in the code.

**Review by weilu — APPROVED** [review]

**Pull request merged**